Library/Formula/open-scene-graph.rb

```diff
@@ -62,7 +62,14 @@ def install
     end
 
     if build.with? "collada-dom"
-      args << "-DCOLLADA_INCLUDE_DIR=#{Formula["collada-dom"].opt_include}/collada-dom"
+      # https://github.com/Homebrew/homebrew/issues/43536
+      collada_include_candidates = Dir.glob("#{Formula["collada-dom"].opt_include}/collada-dom*")
+      if collada_include_candidates.empty?
+        odie "Could not locate collada-dom include directory"
```

Comment:
From the [discussion in one of my PRs](https://github.com/Homebrew/homebrew/pull/44954#discussion_r42487584), I learned it's better to use `raise` instead of `odie`.